docs: remove mention of Teleport Pro from FAQ (#12702)
gravitational/teleport: docs/pages/faq.mdx

---
title: Teleport FAQ

> description: Frequently Asked Questions about using Teleport

Reviewer: ```suggestion
description: Frequently asked questions about using Teleport
```

### Can individual nodes create reverse tunnels to a proxy server without creating a new cluster?

This was a popular customer
[request](https://github.com/gravitational/teleport/issues/803) that was added

> in Teleport version 4.0. Once you've upgraded your Teleport Cluster, change the

Reviewer: ```suggestion
in Teleport version 4.0. Change the
```
Since we no longer support versions <=4.0

> node config option `--auth-server` to point to web proxy address (this would be

Reviewer: ```suggestion
`--auth-server` flag when running the `teleport` daemon on an agent to point to the Proxy Service address (this would be
```
Suggestion to make this a bit more precise and use the Proxy Service product name
